open-machine-learning-jupyter-book/assignments/data-science/displaying-airport-data.ipynb

```diff
@@ -121,9 +121,9 @@
    "outputs": [],
    "source": [
     "%%sql\n",
-    "    SELECT Airports.name, Cities.city, Cities.country\n",
-    "    FROM Airports\n",
-    "    INNER JOIN Cities ON Airports.city_id = Cities.id"
+    "    ______ Airports.name, Cities.city, Cities.country\n",
+    "    ______ Airports\n",
+    "    ______ JOIN Cities ______ Airports.city_id = Cities.id"
```

Comment:
You don't have to remove only the SQL preserved key word. It is also ok to have question for the table name or column name.